sync-core/src/main/java/com/cloudant/sync/datastore/SavedHttpAttachment.java

```diff
@@ -36,7 +36,6 @@
     private URI attachmentURI;
     private int size;
     private byte[] data;
-    private Encoding encoding;
```

Comment:
Okay, I'll open a FB ticket for it so we remember to do it when #143 merges into master